README.md

```diff
@@ -34,7 +34,7 @@ Still not merged:
 2. Clone this repository and cd to it
 3. (Linux < 6.2 only) Run `make older-kernel-patch`
 4. Run `make`
-5. Run `sudo make install`
+5. Run `sudo make dkms-install`
 6. (Optional) To uninstall, run `sudo make uninstall`
```

Comment:
for `dkms-install` you need to run `dkms-uninstall`

Reply:
True! Forgot to add this, just thought I'd include it since it was necessary on my end to replace the currently running msi-ec that came with arch linux 6.7.2.. Using `sudo make dkms-install` passed the injection, while just using `sudo make install` did not pass insmod stage and ultimatly aborted with `modprobe: ERROR: could not insert 'msi_ec': Operation not supported`. 
Thanks for looking into it!

I tested and could verify almost everything:
available_fan_modes, fan_mode, available_shift_modes, shift_mode, battery_mode, cooler_boost, fn_key, win_key, webcam, webcam_block and verified that fw_version and fw_release_date match up with the actual EC release info. 
So far I did not use driver_override; modalias and uevent seem to look good as well.